<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageUI</class>
 <widget class="QWidget" name="ManageUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="list-add"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="list-remove"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="system-reboot"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="document-save"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDropMode::InternalMove</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::DropAction::MoveAction</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okBtn">
         <property name="text">
          <string>&amp;OK</string>
         </property>
         <property name="icon">
          <iconset theme="ok"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeBtn">
         <property name="text">
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
